### PR TITLE
LPAL-1038 fix js issue in feedback form

### PR DESCRIPTION
**Pull request by nickdavis2001**
## Purpose

_Bugfix for feedback form_

Fixes LPAL-1038

## Approach

_Fix logic in js and add extra cypress testing. Tidy the twig template_

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [x] I have added tests to prove my work
* [ ] I have added mandatory tags to terraformed resources, where possible
* [ ] If I have a new OPG component dependency, I have updated the `metadata.json` with the repo location.
* [ ] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

### CODE REVIEW

File: cypress/e2e/Feedback.feature [hunk 1]
```diff
@@ -19,7 +19,16 @@ Feature: Feedback
             | Select a rating for this service |
             | Do not forget to leave your feedback in the box |
         And I see "Error" in the title
-        When I select satisfied
+        # click through all the radio buttons due to history of bug in this area (see LPAL-1038)
```
Comment (townxelliot) [inline cypress/e2e/Feedback.feature:22]: We probably need to do something more than this to prove that the radio button is really being selected. (Clicks on the radio buttons do work, it's just that the value isn't set correctly.) Could be as simple as checking that input on the form; or could intercept the submitted form and check the value of the radio there. The latter would be more complicated but probably more indicative that the form is working properly.

**Review by townxelliot — CHANGES_REQUESTED** [review]

File: cypress/e2e/Feedback.feature (hunk 1 above)
Comment (nickdavis2001) [reply cypress/e2e/Feedback.feature:22]: Done

Comment (townxelliot) [reply cypress/e2e/Feedback.feature:22]: I was expecting some extra assertions that the `rating` input on the form has the expected value. Am I missing something?

Comment (nickdavis2001) [reply cypress/e2e/Feedback.feature]: Whoops. Had committed on wrong git branch. 🤦  Should be sorted now

**Review by townxelliot — APPROVED** [review]

**Pull request merged**